### PR TITLE
Add validation for coach check, fix editor table

### PR DESCRIPTION
**Pull request by ExxZzyy**
Add validation for coach check, fix editor table

### CODE REVIEW

File: src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.html [hunk 1]
```diff
@@ -1,38 +1,62 @@
...
+          min="0"
+          max="10"
+          step="1"
+        />
+      </label>
+      <div
+        *ngIf="form.get('speakingMark')?.invalid && form.get('speakingMark')?.touched"
+        class="validation"
+      >
+        <small *ngIf="form.get('essayMark')?.errors?.pattern">Enter valid score (0-10)</small>
```
Comment (RomPavel) [inline src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.html:47]: should it be 'speakingMark'

File: src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.ts [hunk 2]
```diff
@@ -8,23 +9,36 @@ import { CoachTestsStoreService } from '../service/coach-tests-store.service';
...
     private coachCheck: CoachTestsStoreService,
   ) {}
 
+  form!: FormGroup;
+
+  ngOnInit() {
+    this.form = new FormGroup({
+      comment: new FormControl(''),
+      speakingMark: new FormControl(null, [Validators.pattern('^[0-9]$|[1][0]$')]),
+      essayMark: new FormControl(null, [Validators.pattern('^[0-9]$|[1][0]$')]),
```
Comment (RomPavel) [inline src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.ts:25]: let create constant for regexp

File: src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.html (hunk 1 above)
Comment (ExxZzyy) [reply src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.html]: Done 76032ec330d2896305aef7dc75e6045842596c9c

File: src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.ts (hunk 2 above)
Comment (ExxZzyy) [reply src/app/models/coach-profile/coach-profile-dialog/coach-profile-dialog.component.ts]: Done 76032ec330d2896305aef7dc75e6045842596c9c

**Review by casterweb — APPROVED** [review]

**Review by RomPavel — APPROVED** [review]

**Pull request merged**